examples/main.py

```diff
@@ -0,0 +1,52 @@
+from loggingpy.log import Logger
+import loggingpy.sink as sink
+import time
+import random
+
+
+class InvalidOperationException(Exception):
+    pass
+
+
+if __name__ == "__main__":
+    # Sumologic
+    sumoUri = "https://collectors.de.sumologic.com/receiver/v1/http/ZaVnC4dhaV2bjNoP4SPTO9kxcjdNBNYMhf8MTZci1zG3HYsjnWh4vNCneVeWp7HZGW4cbimAzXDMQLnYTDTfQV9IxAZgLuLcwQD4EA4lWeCk7Ovu1GQpcw=="
+
+    # Logz.io
+    elasticUri = "https://listener.logz.io:8071/?token=itAcKdeCyHzxLpkJMmBDazlEdlIyQvwo&type=dev/test/calculator"
+
+    sumoSink = sink.HttpSink(sumoUri)
+    elasticSink = sink.HttpSink(elasticUri)
+
+    logger = Logger([sumoSink, elasticSink], "Calculator")
+
+    try:
+        for i in range(0, 100000):
+            try:
+                div = random.randint(0, 20)
+                x = 123 / div
+
+                if div == 1:
+                    raise InvalidOperationException("Because I can")
+
+                logger.warning(payload_type="MathOperation", data={
```

Comment:
Is this the "pythonic" way to declare parameters or just due to the order of arguments in the `Logger` API that start with the exception? I'd probably rather do

`logger.warning("MathOperation", { "foo" : "bar" }`

instead of specifying the parameter names.

Reply:
Python has two ways of doing it. Either you just add them as you write it, or you can set the params explicitly and thus have the possibility to reorder the params as you see fit. Additionally, you know exactly what value goes into what parameter in case there are multiple params of the same type (with the python3 type hints, the autocompletion should get better anyway). I can keep or drop this style.

Reply:
ok, let's keep it :)